$NetBSD: patch-ab,v 1.3 2009/09/13 15:09:47 yacht Exp $

In newer versions of openssl, X509V3_EXT_get() now returns a const pointer.
Use a const_cast<>() to be able to build with both versions of the API.

--- talk/base/openssladapter.cc.orig	2008-09-16 14:55:22.000000000 +0100
+++ talk/base/openssladapter.cc
@@ -641,7 +641,7 @@ OpenSSLAdapter::SSLPostConnectionCheck(S
     int extension_nid = OBJ_obj2nid(X509_EXTENSION_get_object(extension));
 
     if (extension_nid == NID_subject_alt_name) {
-      X509V3_EXT_METHOD* meth = X509V3_EXT_get(extension);
+      X509V3_EXT_METHOD* meth = const_cast<X509V3_EXT_METHOD *>(X509V3_EXT_get(extension));
       if (!meth)
         break;
 
